src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import styles from './CourseInfo.module.css';
+
+import Button from '../../common/Button/Button';
+import { mockCourses } from '../../data/data';
+import getCoursesDuration from '../../helpers/getCoursesDuration';
+import getAuthorNames from '../../helpers/getAuthorNames';
+
+const CourseInfo = () => {
+  let { courseId } = useParams();
+  let title;
+  let description;
+  let duration;
+  let authors;
+  let creationDate;
+  mockCourses.forEach((element) => {
+    if (courseId == element.id) {
+      title = element.title;
+      description = element.description;
+      duration = element.duration;
+      authors = element.authors;
+      creationDate = element.creationDate;
+    }
+  });
+
+  const leftArrowUnicode = '\u1438';
+
+  return (
+    <div className={styles.courseInfoForm}>
+      <div className={styles.backToCoursesLink}>
+        <Link to={'/courses'}>
+          <Button buttonText={' ' + leftArrowUnicode + ' Back to courses'} />
```

Comment:
I allow you to use [_Template literals_](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in JS for dynamic string creation, it will simplify your work